https://bugs.gentoo.org/show_bug.cgi?id=612018

From: Thomas Witt <pyromaniac@exherbo.org>
Date: Wed, 15 Mar 2017 20:20:44 +0000 (+0000)
Subject: mount.cifs: Remove data_blob.h include
X-Git-Url: https://git.samba.org/?p=cifs-utils.git;a=commitdiff_plain;h=272d523a57a4e8791d625a479128613be5e401f5

mount.cifs: Remove data_blob.h include

data_blob.h includes talloc.h from libtalloc, but that is only marked as
a dependency for cifs.upcall. No symbols from that header are used by
cifs.mount, so remove it to avoid the libtalloc dependency

Signed-off-by: Thomas Witt <pyromaniac@exherbo.org>
---

diff --git a/mount.cifs.c b/mount.cifs.c
index 13b71ef..2612feb 100644
--- a/mount.cifs.c
+++ b/mount.cifs.c
@@ -61,7 +61,6 @@
 #include "mount.h"
 #include "util.h"
 #include "resolve_host.h"
-#include "data_blob.h"

 #ifndef MS_MOVE
 #define MS_MOVE 8192

